.evergreen/fetch-drivers-tools.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -o errexit
+
+if [[ -z "$DRIVERS_TOOLS" ]]; then
+    echo >&2 "\$DRIVERS_TOOLS must be set"
+    exit 1
+fi
+
+rm -rf $DRIVERS_TOOLS
+git clone https://github.com/mongodb-labs/drivers-evergreen-tools.git $DRIVERS_TOOLS
```

Comment:
this is the snippet in Java driver:
```
  "prepare resources":
    - command: shell.exec
      params:
        shell: bash
        script: |
          ${PREPARE_SHELL}
          rm -rf $DRIVERS_TOOLS
          if [ "${project}" = "drivers-tools" ]; then
            # If this was a patch build, doing a fresh clone would not actually test the patch
            cp -R ${PROJECT_DIRECTORY}/ $DRIVERS_TOOLS
          else
            git clone https://github.com/mongodb-labs/drivers-evergreen-tools.git $DRIVERS_TOOLS
          fi
```
I am puzzled why we need to test `${project}` is equal to `driver-tools` in Java driver. Jeff could explain (seems we need to build a driver-tools patch sometimes), but I don't think we need to consider that in this project.

Reply:
The comment suggests this has something to do with patches, which are fairly common and we will probably need to do in this project, so we should figure out why it is not needed here.

Reply:
I think it might be overkill for hibernate project. For Java driver it might be required to clone a patched `drivers-tools` repo for some reason, but hibernate project only needs very basic features of `driver-tools`. @jyemin , do you think we need to take into consideration of the `drivers-tool` patch build for `mongo-hibernate` project as in Java driver?

Reply:
I do not understand how drivers-tools interacts with this, so I will just leave this open as an "optional" comment. If this is understood to be not necessary, and will not interfere with our ability to run `../evergreen patch ...`, there should be no issue.

Reply:
I think this is all we need.  IIRC we had a template config for all our projects that people just copied, and the template was used for driver-tools itself.  There is no need for that logic either here or in the driver.